bin/asinit

```diff
@@ -132,8 +145,20 @@ rl.question(colors.white("Do you want to proceed?") + " [Y/n] ", answer => {
     "",
     "Have a nice day!"
   ].join("\n"));
-  rl.close();
-});
+}
+
+if (cliOptions.options.yes) {
+  createProject("y");
+} else {
+  const rl = require("readline").createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
+  rl.question(colors.white("Do you want to proceed?") + " [Y/n] ", result => {
+    createProject(result);
+    rl.close();
```

Comment:
Can close before creating the project here

Reply:
Okay.